web_action_conditionable/README.rst

```diff
@@ -40,10 +40,10 @@ To use this module, you need to:
 
```

Comment:
Remove initial sentence, as it's not filled.

Reply:
Instead of removing, you better add usage instructions, because now I don't know what should I test...